src/ts_catalog/catalog.h

```diff
@@ -1173,6 +1174,7 @@ typedef struct FormData_compression_settings
 	ArrayType *orderby;
 	ArrayType *orderby_desc;
 	ArrayType *orderby_nullsfirst;
+	Jsonb *sparse_index;
```

Comment:
This should match the column name in the table
```suggestion
	Jsonb *index;
```